pytest/tests/sanity/rpc_tx_submission.py

```diff
@@ -26,22 +26,21 @@
 
 hash1 = nodes[0].get_latest_block().hash_bytes
 
-for i in range(3):
-    tx = sign_payment_tx(nodes[0].signer_key, 'test1', 100 + i, i + 1, hash1)
-    if i == 0:
-        res = nodes[0].send_tx_and_wait(tx, timeout=20)
-        if 'error' in res:
-            assert False, res
-    else:
-        method_name = 'broadcast_tx_async' if i == 1 else 'EXPERIMENTAL_broadcast_tx_sync'
-        res = nodes[0].json_rpc(method_name,
-                                [base64.b64encode(tx).decode('utf8')])
-        assert 'error' not in res, res
-        time.sleep(5)
-        tx_hash = res['result'] if i == 1 else res['result']['transaction_hash']
-        tx_query_res = nodes[0].json_rpc('tx', [tx_hash, 'test0'])
-        assert 'error' not in tx_query_res, tx_query_res
-    time.sleep(1)
+tx = sign_payment_tx(nodes[0].signer_key, 'test1', 100 + i, i + 1, hash1)
```

Comment:
`i` is undefined

(sounds philisophical :thinking: )

Reply:
Wow, you've found a bigger problem - it looks like we never run this anywhere.
Do you know who's the best person to discuss adding this to our CI?

Reply:
Nayduck runs it, and it failed in the most recent run.
https://nayduck.near.org/#/test/531124
https://github.com/near/nearcore/blob/9e12243b20afc2b1fecf1b8f14d9391c9469d039/nightly/pytest-sanity.txt#L118

Reply:
Fixed issue with `i`